### ISSUE TITLE
Add Patient Name to Results DOB Entry Screen

### ISSUE THREAD

**Issue by tim-skylight** [CDCgov/prime-simplereport#3184]
### WHY?
Based on a high volume of support tickets related to expired results links, we believe there is some confusion around the birthdate that must be entered to acccess a result. We would like to make this more clear by including the name of the patient on the result, in an effort to make it clear whos birthdate should be entered to access a result

**AS A** receiver of SimpleReport results emails

**I WANT** it to be clear who the result i receive pertains to

**SO THAT** i know the DOB to enter in order to access my result

---

**GIVEN** a result has been sent to me

**WHEN** I follow the link in the result email and reach the screen where i have to authenticate with a DOB

**THEN** I can see the name of the person the result is for 

**BEFORE:** Enter 'your date of birth' to access your COVID-19 testing portal.
**AFTER:** Enter 'the date of birth provided for insert patient name' to access your COVID-19 testing portal

Note: this link will expire 5 days after the test result was recorded.

Date of birth
For example: 4 28 1986
Month
Day
Year
Continue

**Comment by tim-skylight** [comment]
Whoever picks this up, please pull in @vz3 to pair

**Comment by LGFranklin** [comment]
Please contact the support team as soon as designs are live so that they know what the new version will look like.

They want to know where the name is going (on email/ text AND on the link page or just on the link page?)...

Also, there is a recommendation as we're editing this to remove SimpleReport from the intro on the email that starts with "Greetings from SimpleReport" either to "Greetings" or "Greetings from FACILITY" to reduce the impression that folks should come to use for follo-wup

**Comment by Rebecca-LM** [comment]
@nathancrtr please see draft copy suggested [in the body of this doc](https://docs.google.com/document/d/1XBcBnS-kjXVbT44q7RN_6Nx9QmdFr-aNFj_IjaV3UhE/edit#). Are we able to customize the name of the facility and expiration date of results here? We created copy with the assumption that we can, lmk if not and I'll modify.

**Comment by nathancrtr** [comment]
@Rebecca-LM yes, we can add the facility name and link expiration date to the DOB verification screen and it's a great idea! There will be a bit of lift involved since that information is not available to the application in advance. It will have to get that data from the server, which adds a bit of complexity.

At this point I would lean towards breaking out facility name+expiration date into a new ticket, @vz3 how does that sound?

**Comment by LGFranklin** [comment]
I'm not @vz3 :) but I agree with that @nathancrtr - let's ship without it on this first go since the most requested info is the name and facility and folks can do the math with the time stamp and # days in the meantime.

**Comment by vz3** [comment]
I'm good w breaking those into a new story

**Comment by Rebecca-LM** [comment]
Great! @nathancrtr are you able to create the new story? Just want to make sure it gets logged/addressed when everyone has time to loop back :)

**Comment by nathancrtr** [comment]
@Rebecca-LM yes indeed, see the new story here: https://app.zenhub.com/workspaces/simplereport-better-data-team-605b43d51ea9f700119a48ee/issues/cdcgov/prime-simplereport/3332

**Comment by Rebecca-LM** [comment]
much appreciated! lmk if you need my eyes on things
